README.md

```diff
@@ -47,21 +47,10 @@ If your target environment does not support that, make sure you transpile this p
 1. [Node 14](https://nodejs.org/en/download/) ([nvm](https://github.com/nvm-sh/nvm) or [fnm](https://github.com/Schniz/fnm) recommended)
 2. [Yarn](https://classic.yarnpkg.com/en/docs/install/)
 
-### Commit messages and releasing to NPM
+### Releases
 
-_tl;dr_
+Commits that are merged to `main` on GitHub will be analysed by [semantic-release](https://semantic-release.gitbook.io/).
 
--   prefer `yarn commit` to `git commit`
--   continuous delivery to NPM
+If the commit messages follow the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0), semantic-release will determine what types of changes are included.
```

Comment:
```suggestion
If the PR title follows the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0), semantic-release will determine what types of changes are included.
```

Reply:
yeah this is tricky. technically it _is_ the commits themselves – the PR title becomes the commit message when the branch is merged. github is just squashing the other commits for you... will have another look at the wording

Reply:
had another go